Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,59 @@
+//Solved by Arion Hamel
+#include "Person.h"
+#include <iostream>
+#include <string>
+#include <cstdlib>
+
+// -- Constructors --
+Person::Person() {
+    name_ = "";
+    age_ = new int(0);
+}
+
+Person::Person(const std::string& name, int age) {

```

Comment:
Try to use initializer lists instead:
`Person::Person(const std::string& name, int age): name_(name), age_(new int(age)){}`